std/net/curl.d

```diff
@@ -627,6 +627,61 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
+ */
+private string urlencode()(in string[string] values)
+{
+    bool first = true;
+    Appender!string enc;
+    enc.reserve(values.length * 512);
+    foreach (k, v; values)
+    {
+        if (!first)
+            enc ~= '&';
+        import std.uri;
```

Comment:
As a matter of style, the local imports must be selective: `import std.uri: encodeComponent;`. This should have been detected by Travis-CI dscanning phase.